tests/acceptance/basic-test.js

```diff
@@ -27,26 +27,12 @@ module('Acceptance: modal-dialog | no animation, no tether', function (hooks) {
       openSelector: '#example-basic button',
       dialogText: 'Basic',
       closeSelector: overlaySelector,
-      whileOpen: async function () {
-        assert.hasDataTest(
-          dialogSelector,
-          'my-data-test',
```

Comment:
I removed the assertions with `my-data-test`, because they, in essence, test if Ember's splattributes work.